.github/workflows/release-please.yaml

```
@@ -6,13 +6,66 @@
+
+        - name: Build and push Docker image with version tag
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: ./docker/Dockerfile
+            push: true
+            tags: |
+                ${{ secrets.DOCKER_HUB_USERNAME }}/s-private:latest
+                ${{ secrets.DOCKER_HUB_USERNAME }}/s-private:${{ env.VERSION }}
+            secrets:
+              AUTH_SECRET: ${{secrets.AUTH_SECRET}}
```

actionlint (1.7.4)

expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)